src/components/Elixir/AddLiquidity/index.tsx

```diff
@@ -443,7 +443,13 @@ const AddLiquidity: React.FC<AddLiquidityProps> = (props) => {
               )}
             </DynamicSection>
 
-            <DynamicSection disabled={!feeAmount || invalidPool || (noLiquidity && !startPriceTypedValue)}>
+            <DynamicSection
+              disabled={
+                !feeAmount ||
+                invalidPool ||
+                (noLiquidity && (!startPriceTypedValue || parseFloat(startPriceTypedValue) === 0))
```

Comment:
This conditional is quite hard to read. Consider moving it to the function which describes it by name :)